leaper.py

```diff
@@ -225,6 +225,27 @@ def check_source_submission(self, src_project, src_package, src_rev, target_proj
 
             return review_result
 
+        if target_project.endswith(':Update'):
+            self.logger.info("expected origin is '%s' (%s)", origin,
+                             "unchanged" if origin_same else "changed")
+
+            if origin_same:
+                return True
+
+            good = self._check_matching_srcmd5(origin, target_package, src_srcinfo.verifymd5)
+            if good:
+                self.logger.info('submission source found in origin ({})'.format(origin))
+                return good
+            good = self.factory._check_requests(origin, target_package, src_srcinfo.verifymd5)
+            if good or good == None:
+                self.logger.info('found pending submission against origin ({})'.format(origin))
+                return good
+
+            return None
+        elif self.action.type == 'maintenance_incident':
+            self.logger.debug('incident not yet asigned a target release project')
```

Comment:
does that happen in practice?

Reply:
No it does not. I was thinking about the silly `target_package` not being set until it is assigned to an incident.